helm-charts/dnsbl-exporter/values.yaml

```diff
@@ -0,0 +1,339 @@
+## @param replicaCount Number of Aplication replicas to deploy
+## NOTE: ReadWriteMany PVC(s) are required if replicaCount > 1
+##
+replicaCount: 1
+## @param updateStrategy.type Application deployment strategy type
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+## NOTE: Set it to `Recreate` if you use a PV that cannot be mounted on multiple pods
+## e.g:
+## updateStrategy:
+##  type: RollingUpdate
+##  rollingUpdate:
+##    maxSurge: 25%
+##    maxUnavailable: 25%
+##
+updateStrategy:
+  type: RollingUpdate
+## @param priorityClassName powerdnsAdmin pod priority class name
+##
+priorityClassName: ""
+## @param schedulerName Name of the k8s scheduler (other than default)
+## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+##
+schedulerName: ""
+## @param topologySpreadConstraints Topology Spread Constraints for pod assignment
+## https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+## The value is evaluated as a template
+##
+topologySpreadConstraints: []
+## @param hostAliases Application pod host aliases
+## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
+##
+hostAliases: []
+## @param extraVolumes Optionally specify extra list of additional volumes for powerdnsAdmin pods
+##
+extraVolumes: []
+## @param extraVolumeMounts Optionally specify extra list of additional volumeMounts for powerdnsAdmin container(s)
+##
+extraVolumeMounts: []
+## @param sidecars Add additional sidecar containers to the powerdnsAdmin pod
+## e.g:
+## sidecars:
+##   - name: your-image-name
+##     image: your-image
+##     imagePullPolicy: Always
+##     ports:
+##       - name: portname
+##         containerPort: 1234
+##
+sidecars: []
```

Comment:
This could also go? Unless we plan to add an unbound sidecar, or something?

Reply:
Very helpful for debugging (like permissions, limitations, etc in the cluster). Personally I added a sidecar with ubuntu/alpine to ping/trace/connect something.

Reply:
@zentavr can you explain what the sidecar is used for, for the exporter? I think if people want extra tooling, this can be maintained outside of this.

Reply:
Saying you can define a sidecar with unbound if you really need that.

without having that block it becomes more with more pain like monkey-patching with the tools like helm hooks or Kustomize.

by default it is not defined, so not used. But easily could be enabled by few lines of code.